packages/compartment-mapper/src/parse-archive-cjs.js

```diff
@@ -39,6 +43,11 @@ export const parseArchiveCjs = async (
   return {
     parser: 'pre-cjs-json',
     bytes: pre,
-    record: freeze({ imports, exports, reexports }),
+    record: /** @type {ThirdPartyStaticModuleInterface} */ (freeze({
+      imports,
+      exports,
+      reexports,
+      execute: noopExecute,
```

Comment:
Since I'm not qualified yet to review any runtime changes, I assume `execute` was never used before but required in types?

Reply:
Correct. We don’t execute a module when it’s on its way into a Zip file.